packages/react-reconciler/src/__tests__/ReactSuspense-test.internal.js

```diff
@@ -271,4 +272,49 @@ describe('ReactSuspense', () => {
     expect(ReactTestRenderer).toHaveYielded(['Hi', 'Did mount: Hi']);
     expect(root).toMatchRenderedOutput('Hi');
   });
+
+  it('only captures if `fallback` is defined', () => {
+    const root = ReactTestRenderer.create(
+      <Suspense fallback={<Text text="Loading..." />}>
+        <Suspense maxDuration={100}>
+          <AsyncText text="Hi" ms={5000} />
+        </Suspense>
+      </Suspense>,
+      {
+        unstable_isConcurrent: true,
+      },
+    );
+
+    expect(root).toFlushAndYield([
+      'Suspend! [Hi]',
+      // The outer fallback should be rendered, because the inner one does not
+      // have a `fallback` prop
+      'Loading...',
+    ]);
+    jest.advanceTimersByTime(1000);
+    expect(ReactTestRenderer).toHaveYielded([]);
+    expect(root).toFlushAndYield([]);
+    expect(root).toMatchRenderedOutput('Loading...');
+
+    jest.advanceTimersByTime(5000);
+    expect(ReactTestRenderer).toHaveYielded(['Promise resolved [Hi]']);
+    expect(root).toFlushAndYield(['Hi']);
+    expect(root).toMatchRenderedOutput('Hi');
+  });
```

Comment:
This test case is weird, I don't think we need provide this usage of Suspense(at lease for this case behavior). In this case, we set the `maxDuration` option  but it is useless, seems we use the `maxDuration` of the outer Suspense(default `0`?), this is not expected.

So just from my personal perspective, we have 2 ways:
1. Warn about this usage if we allow to do this

or 

2. Use the `maxDuration`(if provided) of inner and `fallback` of outer